assets/base.css

```diff
@@ -2462,3 +2462,46 @@ details-disclosure > details {
   height: 0;
   padding-bottom: var(--ratio-percent);
 }
+
+.content-containers {
+  border-radius: var(--text-boxes-radius);
```

Comment:
I think eventually we should remain these variables to something like `--content-containers-radius` for consistency, but we can do that after this is merged